### PR TITLE
[google_maps_flutter] Crash of moveCamera on old android devices

### PR DESCRIPTION
**Pull request by EdSabino**
## Description

Correct a crash bug on old android devices.
If you pass determinate numbers to LatLng(), and use moveCamera, the app just crash, everything point to a problem of floating point arithmetic, i have this problem with the following latLng:
LatLng(-27.1037217, -52.61875810000001).

I make this bug happens on a Pixi3, with android 5.1, updating the map right after he is completed rendered.

## Related Issues

I don`t find a relate issue about that

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. [shared_preferences]
- [x] I updated pubspec.yaml with an appropriate new version according to the [pub versioning philosophy].
- [x] I updated CHANGELOG.md to add a description of the change.
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://www.dartlang.org/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

**Comment by otopba** [comment]
@EdSabino 
Hi!
Could you provide logs with crash? Thank you

**Comment by cyanglaz** [comment]
@EdSabino Could you create a Github issue describing the issue and link this PR to it?

File: packages/google_maps_flutter/lib/src/location.dart [hunk 1]
```diff
@@ -26,8 +26,9 @@ class LatLng {
   /// The longitude in degrees between -180.0 (inclusive) and 180.0 (exclusive).
   final double longitude;
 
+  /// The latitude and longitude must be adjusted because they can, in old devices, com as a double with too many digits after the dot, and, some times, the java plugin crashes because of it
   dynamic _toJson() {
-    return <double>[latitude, longitude];
+    return <double>[double.parse(latitude.toStringAsFixed(2)), double.parse(longitude.toStringAsFixed(2))];
```
Comment (cyanglaz) [inline packages/google_maps_flutter/lib/src/location.dart:31]: AFAIK, lat and long are 15 decimal digits right from the decimal point. We shouldn't just round to 2 digits.
If it only affects certain old android/java version, we should indicate that in the code as well. like 
```
if version < xxx {
   // do the rounding
}
```

**Comment by cyanglaz** [comment]
@EdSabino Friendly ping :) it's been a while, do you still plan to work on it?

**Comment by cyanglaz** [comment]
There doesn't seem to be any diff of this PR. I'm going to close this.

**Pull request closed without merging**